examples/Heat/mms.py

```diff
@@ -0,0 +1,200 @@
+from math import pi
```

Comment:
This is really great to have an MMS example!

It would be good to turn it into a test as well, so that the time-explicit parts of the code have some test coverage.
`test_steady_swe` in `tests/test_paradiag.py` does a similar thing of checking the paradiag solution against a known solution (in that test checking that it maintains a steady state condition) so would be a good test to look at to see what the MMS test would need to do.

Reply:
I see this is now in `test_Nitsche_BCs`, thanks!